File: app/src/main/assets/templates/ViewBinding Project/java/app/src/main/java/$packagename/MainActivity.java
package $packagename;

import $packagename.databinding.ActivityMainBinding;
import android.os.Bundle;
import android.view.View;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

  private ActivityMainBinding binding;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    binding = ActivityMainBinding.inflate(getLayoutInflater());
    View view = binding.getRoot();
    setContentView(view);
  }
}

File: app/src/main/assets/templates/ViewBinding Project/kotlin/app/src/main/java/$packagename/MainActivity.kt
package $packagename

import androidx.appcompat.app.AppCompatActivity
import $packagename.databinding.ActivityMainBinding
import android.os.Bundle

public class MainActivity : AppCompatActivity() {

	private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         val view = binding.root
         setContentView(view)
    }
}
